system/jlib/jstats.cpp

```diff
@@ -1236,7 +1206,18 @@ void StatisticsMapping::createMappings()
     }
 }
 
-const StatisticsMapping allStatistics;
+
+class AllStatisticsMapping : public StatisticsMapping
+{
+public:
+    AllStatisticsMapping()
+    {
+        for (int i = StKindAll+1; i < StMax; i++)
+            indexToKind.append(i);
+        createMappings();
+    }
+} allStatisticsMapping;
+const StatisticsMapping &allStatistics = allStatisticsMapping;
```

Comment:
What is the reason for this change?

Reply:
It's related to the changes in the constructor of StatisticsMapping.
Before the default constructor was the 'all' types constructor.
With the variadic list on the constructor, the default constructor had to be nop.